modules/contact/src/actions/ContactAction.C

```diff
@@ -1028,6 +1071,129 @@ ContactAction::addNodeFaceContact()
   }
 }
 
+void
+ContactAction::createSidesetPairsFromGeometry()
+{
+  mooseInfo("The contact action is reading the list of sidesets and matching them assuming that: "
+            "1) Those sidesets wrap around the surfaces of the bodies, 2) Axisymmetry of the "
```

Comment:
Is axisymmetry really the right term here? Seems like something like "periodically repeating geometry" is more appropriate.